hw_12/index.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', 1);
+
+//Task 1
+
+function sayHi(): string
+{
+    return "Hello, World!";
+}
+
+//Task 2
+
+function getYear(int $year): bool
+{
+    if ($year % 4 === 0) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
if лишний
операция сравнения и так вернут bool
можно сделать сразу
```suggestion
    return $year % 4 === 0;
```